tests/host_tools/mmio_write.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <sys/mman.h>
+#include <stdint.h>
+#include <fcntl.h>
+
+int main(int argc, char* argv[])
+{
+  int fd;
+  unsigned char *p;
+  int ret = 0;
+
+  fd = open("/dev/mem", O_RDWR);
+  if (fd < 0)
+    {
+      perror("Failed to open /dev/mem");
+      return 1;
+    }
+
+  p = mmap(NULL, 0x2000, PROT_READ|PROT_WRITE, MAP_SHARED, fd, 0xd0000000);
+  if (p == MAP_FAILED)
+    {
+      perror("Failed to mmap /dev/mem");
+      return 2;
+    }
+
+  p[0x100]+=8;
```

Comment:
Please add a comment explaining that this triggers an MMIO config space rewrite.

Reply:
Done.